core/templates/dev/head/editor/exploration_history.html

```diff
@@ -18,24 +18,23 @@
     <h3>List of Changes</h3>
     <div ng-repeat="versionNumber in snapshotOrderArray" class="row">
```

Comment:
I think it is now an object representing a checkbox, so I would suggest calling it versionCheckbox and versionCheckboxArray